draft-ietf-core-oscore-groupcomm.md

```diff
@@ -1840,17 +1834,27 @@ Note that the Partial IV of an endpoint does not necessarily grow monotonically.
 
 Since one-to-many communication such as multicast usually involves unreliable transports, the simplification of the Replay Window to a size of 1 suggested in {{Section 7.4 of RFC8613}} is not viable with Group OSCORE, unless exchanges in the group rely only on unicast messages.
 
-A server may not be synchronized with a the client's Sender Sequence Number, or a Replay Window may be initialized as not valid, see {{ssec-loss-mutable-context}}. {{ssec-loss-mutable-context-total}} and {{ssec-loss-mutable-context-overflow}} define measures that endpoints need to take in such situations, before resuming to accept incoming messages from other group members.
+A server may not be synchronized with a the client's Sender Sequence Number, or a Replay Window may be initialized as invalid, see {{ssec-loss-mutable-context}}. The server can either retrieve a new security context or synchronize the sequence numbers before resuming to accept incoming messages from other group members.
+
+## Message Ordering {#ssec-seccons-ordering}
+
+Assuming the other endpoint is honest, Group OSCORE provides relative ordering of received messages. For a given Security Context, the received Partial IV (when included) allows the recipient endpoint to determine the order in which requests or responses were sent by the other endpoint. In case the Partial IV was omitted in a response, this indicates that it was the oldest response to corresponding request (like notification responses in OSCORE, see {{Section 7.4.1 of RFC8613}}). A received response is not older than the corresponding request.
```

Comment:
`Assuming that the other endpoint is honest, Group OSCORE provides relative ordering of received messages. For a given Group OSCORE Security Context, the received Partial IV (when included) allows the recipient endpoint to determine the order in which requests or responses were sent by the other endpoint. In case the Partial IV was omitted in a response, this indicates that it was the oldest response from the sender endpoint to the corresponding request (like notification responses in OSCORE, see {{Section 7.4.1 of RFC8613}}). A received response is not older than the corresponding request.`